libr/core/cmd_open.c

```diff
@@ -375,6 +375,33 @@ static void cmd_open_bin(RCore *core, const char *input) {
 			}
 		}
 		break;
+	case '=': // "ob="
+		{
+			RListIter *iter;
+			RList *list = r_list_new ();
+			RBinFile *binfile = NULL;
```

Comment:
Use bf instead of binfile. its shorter and im doing this change in all the codebase